adr/util/config.py

```diff
@@ -0,0 +1,42 @@
+import yaml
+
+
+class Configuration(object):
+    def __init__(self, url):
+        with open(url, 'r') as ymlfile:
+            self.cfg = yaml.load(ymlfile)
+
+    @property
```

Comment:
Rather than defining `@property` for each variable, this would be a good use case for slots:
https://docs.python.org/2/reference/datamodel.html#slots

Basically you define which attributes you expect the config object to have in `__slots__` and then it'll be an error if any of them are missing/extra ones after instantiation. Here is an example of `__slots__`:
https://searchfox.org/mozilla-central/source/python/mozlint/mozlint/result.py#68

If you do this, you'll be able to read the `.yml` file in `__init__` like normal, and then dynamically add each config value to the class. If a value is wrong, then python will error.